specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -4060,6 +4121,38 @@
         }
       }
     },
+    "BillingProfileCreationParameters": {
+      "description": "The parameters for creating a new billing profile.",
+      "properties": {
+        "displayName": {
+          "description": "The billing profile name.",
+          "type": "string"
+        },
+        "poNumber": {
+          "description": "Purchase order number.",
+          "type": "string"
+        },
+        "address": {
+          "description": "Billing address.",
+          "$ref": "#/definitions/Address"
+        },
+        "invoiceEmailOptIn": {
```

Comment:
bools are not recommended. please make this an enum, the only supported value for not would be email. the property can be called invoiceOptIn or invoiceDeliveryMethod etc. how do you get the email address?

Reply:
This is leverage the existing property of the billing profile. In the next api version, we can switch out all the bools to be enums.

Reply:
How do you get the email address to which the notification needs to be sent?

Reply:
there must already be an existing billing profile for this billing account that has a customer associated with it. This new billing profile is being created for the purpose for creating a new invoice for the same customer and will be sent to the same email address as the existing billing profile. The scenario is that a customer wants to create separate invoices that get sent to the same email.